src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -2928,6 +2936,45 @@ checkNullable:
             Return typeName
         End Function
 
+        Private Function ParseTupleType(openParen As PunctuationSyntax) As TypeSyntax
```

Comment:
Note that  `()` will also parse as a tuple type, but will complain about missing type. Similarly to how `()` in expression context parses as parenthesized expression but would complain about missing expression.